shenyu-client/shenyu-client-common/src/main/java/org/apache/shenyu/client/common/exception/ShenyuClientException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.client.common.exception;
+
+/**
+ * ShenyuClientException.
+ */
+public class ShenyuClientException extends IllegalArgumentException {
```

Comment:
why it extend IllegalArgumentException?

Reply:
I see the client just validating the arguments.

Reply:
I see the client just validate the arguments.

Reply:
so why not change the name to `ShenyuIllegalException` ? And I think you can move them in `shenyu-client-core`, no need to creaye a new module like client-common.

Reply:
did it. use `ShenyuClientIllegalArgumentException`.